webapp/src/cljc/lipas/schema/core.cljc

```diff
@@ -321,16 +321,27 @@
           :opt-un [:lipas.location.city/neighborhood]))
 
 ;; -90-90
-(s/def :lipas.location.coordinates/lat (double-in :min 59.846373196
-                                                  :max 70.1641930203
+(s/def :lipas.location.coordinates/lat (double-in :min -90
+                                                  :max 90
                                                   :NaN? false
                                                   :infinite? false))
 ;; -180-180
-(s/def :lipas.location.coordinates/lon (double-in :min 20.6455928891
-                                                  :max 31.5160921567
+(s/def :lipas.location.coordinates/lon (double-in :min -180
+                                                  :max 180
                                                   :NaN? false
                                                   :infinite? false))
 
+;; -90-90
```

Comment:
I think we should mostly restrict to coords within Finland's bounds. Other coordinates make little sense in most situations and where they make sense, it's more likely an exception than a rule. Or what do you think?